rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml

```
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies desktopimgdownldr utility being used to download a remote file. Verify details such as parent process, url
+reputation, and downloaded file details.
```

```suggestion
Identifies the desktopimgdownldr utility being used to download a remote file. An adversary may use desktopimgdownldr to download arbitrary files as an alternative to certutil. 
```
move to `note` - Verify details such as parent process name, URL reputation, and downloaded file details.

---

may need to re-lint if accepted